google/downloader.sh

```diff
@@ -6,58 +6,100 @@
 # From: https://gist.github.com/jeffmccune/e7d635116f25bc7e12b2a19efbafcdf8
 # From: https://gist.github.com/n0531m/f3714f6ad6ef738a3b0a
 
+# Script to retrieve and organize Google and Google Cloud IP ranges.
+
 set -euo pipefail
 set -x
 
+# Check for required dependencies
+for cmd in curl dig jq mktemp; do
+  if ! command -v "$cmd" &> /dev/null; then
+    echo "Error: $cmd is not installed or not in PATH" >&2
+    exit 1
+  fi
+done
+
+# Create a temporary directory and ensure cleanup on exit
```

Comment:
I think its not need:
- `/tmp` availiable every time 
- worker live less 1 minute